### DISCUSSION TITLE
xxl alias for 2xl breakpoint in attributify mode

### DISCUSSION THREAD

**Discussion by toniengelhardt** [unocss/unocss#2876] [Ideas]
In principle I prefer the 2xl, 3xl, etc. syntax over xxl, xxxl, ..., e.g. `text-4xl` and so forth. But there is a problem with breakpoints in  attributify mode where `2xl:something` doesn't work (probably bc it cannot start with a number?). The workaround is to use `class="2xl:something"`, which works, but is not very elegant.

My suggestion would be to add xxl as alias for 2xl, xxxl as alias for 3xl, etc. only for the default breakpoints if that is possible.

**Comment by userquin** [comment]
Makes sense
